userLoggedInProfile.php

```diff
@@ -0,0 +1,163 @@
+echo $profileEdit;  */
+
+
+
+
+if(isset($_REQUEST['happinessQuestionnaireInput'])){
+    insertNewHappinessQuestionnaireInput(
+        $_REQUEST['email'],
+        $_REQUEST['happinessQuestion1'],
+        $_REQUEST['happinessQuestion2'],
+        $_REQUEST['happinessQuestion3'],
+        $_REQUEST['happinessQuestion4']
+       
+     
+        );
+    }
+
+?>
+
+
+
+
+<html>
+  <h2>Profile</h2>
+    <form action='' method='post'>  
+            Email:
+            <input type='text' name='email' value= '<?php echo $user['email'] ?> '  /><br />
+
+            Age:
+            <input type='text' name='age' value= '<?php echo $user['age'] ?>' /><br />
+
+            Gender:
+            <input type='text' name='gender' value= '<?php echo $user['gender'] ?>'/><br />
+            
+            Occupation:
+            <input type='text' name='occupation' value= '<?php echo $user['occupation'] ?>'/><br />
+
+            Occupation Stress Level:
+                <select name='stressLevel'>
+                    <option value='low'>little to no stress</option>
+                    <option value='medium'>some/moderate stress</option>
+                    <option value='high'>High stress</option>
+                </select> <br /> 
+
+            What kind of caffeinated beverage do you typically drink:
+                <select name='typeBeverage'>
+                    <option value='coffee'>coffee</option>
+                    <option value='tea'>tea</option>
+                    <option value='soda'>soda</option>
+                    <option value='other'>other</option>
+                </select><br /> 
+            
+            Please specify what kind of beverage (ie black coffee, cola, green tea, etc):
+                <input type='text' name='beverageOther' value= '<?php echo $user['beverageOther'] ?>'/> <br />
+
+
+            Have you recently experienced or are currently experiencing a life altering event? (ie moving houses, switching jobs, etc )?<br />
+            <input type="radio" name="lifeEvent" value='yes' > yes
+            <input type="radio" name='lifeEvent' value='no'> no<br>
+
+
+            <br/><br/>
+            <a href='editProfilePage.php'>edit profile page</a>
+            <br/><br/>
+           
+            Go to the <a href='questionnaireForm.php'>daily questionnaire</a>
+            
+            <br /><br />
+            <br />
+
+            <h2>Happiness Scale</h2>
+            <p> 
+            Please fill out this questionnaire after filling out the last daily questionnaire. </p>
+            <br />
+         
+        
+            Email:
+            <input type='text' name='email' value= '<?php echo $user['email'] ?> '  /><br />
+
+
+           <p>In general I consider myself  </p>
+            <input type="radio" name="happinessQuestion1" value="1" >1 (not a very happy person)
+           <input type="radio" name="happinessQuestion1" value="2" >2
```

Comment:
Once again, maybe not worth changing now, but naming this `happinessQuestion1` seems pretty vague, and storing it in the user table instead of in it's own table seems a bit suboptimal (unless you really only want people to answer it once, and you don't want access to changes over time)